tests/jobs/test_scheduler_job.py

```diff
@@ -2087,7 +2087,7 @@ def run_with_error(task):
             except AirflowException:
                 pass
 
-        ti_tuple = six.next(six.itervalues(executor.queued_tasks))
+        ti_tuple = list(executor.queued_tasks.values())[0]
```

Comment:
I think this is not the best way of replacing six.next. The original code did not build the whole list - just an iterator and then took the first element of the iterator. In the proposed solution we are building the whole list of values and take the first element from that list. The list might be pretty long if you have a long queue of tasks so converting to list might change performance characteristics of this piece of code. Not sure if that matters but I think we should be careful with that. Python3 next is much more obvious choice:

```
next(iter(executor.queued_tasks.values()))
``` 

values() is a view object, iter builds an iterator to access it and next takes the first element from the iterator.

Reply:
Thanks. I updated this PR.